helm-repos/index.yaml

```diff
@@ -1,11 +1,48 @@
 apiVersion: v1
 entries:
   kuberhealthy:
+  - apiVersion: v2
+    appVersion: eric-stuff
```

Comment:
This was the name of the branch being merged.  This should have been the tag that was created.